shared/validate/validate.go

```diff
@@ -884,3 +884,12 @@ func IsMultipleOfUnit(unit string) func(value string) error {
 		return nil
 	}
 }
+
+// IsStaticCPUPinning validates a static CPU pinning strategy.
+func IsStaticCPUPinning(value string) (static bool, err error) {
```

Comment:
No need to return bool and error, just error is in keeping with the format of the other functions, and a nil error can mean "is static CPU pinning"